synapse/storage/schema/delta/38/profile.py

```diff
@@ -0,0 +1,104 @@
+# Copyright 2016 OpenMarket Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from synapse.storage.prepare_database import get_statements
+from synapse.storage.engines import PostgresEngine
+
+import logging
+import ujson
+
+logger = logging.getLogger(__name__)
+
+CREATE_TABLE = """
+CREATE TABLE profiles_extended (
+    stream_id BIGINT NOT NULL,
+    user_id TEXT NOT NULL,
+    persona TEXT NOT NULL,
+    key TEXT NOT NULL,
+    content TEXT NOT NULL
+);
```

Comment:
Maybe comment on what the fields are?
Is the "content" JSON? Maybe comment on whether it is or not.